src/Cli/func/Actions/LocalActions/PackAction/CustomPackSubcommandAction.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Azure.Functions.Cli.Interfaces;
+using Fclp;
+
+namespace Azure.Functions.Cli.Actions.LocalActions.PackAction
+{
+    [Action(Name = "pack custom", ParentCommandName = "pack", ShowInHelp = false, HelpText = "Arguments specific to custom worker runtime apps when running func pack")]
+    internal class CustomPackSubcommandAction : PackSubcommandAction
+    {
+        public override ICommandLineParserResult ParseArgs(string[] args)
+        {
+            return base.ParseArgs(args);
+        }
+
+        public async Task RunAsync(PackOptions packOptions)
+        {
+            await ExecuteAsync(packOptions);
+        }
+
+        protected override Task<string> GetPackingRootAsync(string functionAppRoot, PackOptions options)
+        {
```

Comment:
We should get the path for custom handler executable from host.json and add a warning in case the executable is not present in that directory.